### PR TITLE
Add a new song called War Cant of Mars

### PR DESCRIPTION
**Pull request by AlchemicRonin**
[War Cant of Mars](https://www.youtube.com/watch?v=wy-sVTaZRPk)

### CODE REVIEW

File: examples/buzzer/main.cc [hunk 1]
```diff
@@ -25,25 +25,48 @@
...
-  { Note::So5L,   80  },
-  { Note::Silent, 560 },
-  { Note::Finish, 0 },
+static bsp::BuzzerNoteDelayed Mario[] = {
+    {Note::Mi3M, 80},    {Note::Silent, 80}, {Note::Mi3M, 80},
+    {Note::Silent, 240}, {Note::Mi3M, 80},   {Note::Silent, 240},
+    {Note::Do1M, 80},    {Note::Silent, 80}, {Note::Mi3M, 80},
+    {Note::Silent, 240}, {Note::So5M, 80},   {Note::Silent, 560},
+    {Note::So5L, 80},
+};
```
Comment (AlchemicRonin) [inline examples/buzzer/main.cc:34]: I use Clang-Format to format my code into Google style, so you may see that there are some pieces of code formatted automatically by Clang-Format into Google style.

Comment (alvinsunyixiao) [reply examples/buzzer/main.cc:34]: That is okay for now, but try not to format things as we have not yet come to an agreement on which format we should follow yet.

File: examples/buzzer/main.cc [hunk 2]
```diff
@@ -25,25 +25,48 @@
...
+    {Note::Re2H, 200},   {Note::Mi3H, 1200}, {Note::Do1H, 800},
+    //    {Note::Silent, 400}, {Note::La6M, 800},   {Note::Si7M, 400},
+    //    {Note::Do1H, 400},   {Note::Si7M, 200},   {Note::Do1H, 400},
+    //    {Note::La6M, 200},   {Note::So5M, 1600},  {Note::Mi3M, 800},
+    //    {Note::Silent, 400}, {Note::So5M, 800},   {Note::Fa4M, 400},
+    //    {Note::Mi3M, 400},   {Note::So5M, 200},   {Note::Do1H, 400},
+    //    {Note::Re2H, 200},   {Note::Mi3H, 1600},  {Note::Do1H, 800},
+    //    {Note::Do1H, 800},   {Note::Re2H, 800},   {Note::Re2H, 800},
+    //    {Note::Do1H, 800},   {Note::Si7M, 800},   {Note::Do1H, 1600},
+    //    {Note::Silent, 800}, {Note::Silent, 800}, {Note::Finish, 0},
```
Comment (alvinsunyixiao) [inline examples/buzzer/main.cc:65]: what are all these commented lines for?

Comment (AlchemicRonin) [reply examples/buzzer/main.cc:65]: This is a whole song. But I think the song is so long that most people may prefer to delete it. So I only pick one sentence from it and comment all their parts.

File: examples/buzzer/main.cc (hunk 1 above)
Comment (AlchemicRonin) [reply examples/buzzer/main.cc:34]: Ok. I already tested the code. It functions well. But I will not format any code next time.

File: examples/buzzer/main.cc [hunk 3]
```diff
@@ -25,25 +25,48 @@
...
-  { Note::Silent, 560 },
-  { Note::Finish, 0 },
+static bsp::BuzzerNoteDelayed Mario[] = {
+    {Note::Mi3M, 80},    {Note::Silent, 80}, {Note::Mi3M, 80},
+    {Note::Silent, 240}, {Note::Mi3M, 80},   {Note::Silent, 240},
+    {Note::Do1M, 80},    {Note::Silent, 80}, {Note::Mi3M, 80},
+    {Note::Silent, 240}, {Note::So5M, 80},   {Note::Silent, 560},
+    {Note::So5L, 80},
+};
+
```
Comment (alvinsunyixiao) [inline examples/buzzer/main.cc:35]:
```suggestion
// uncomment to play the full song
```

**Pull request merged**